internal/xds/translator/listener.go

```diff
@@ -137,6 +163,10 @@ func (t *Translator) addXdsHTTPFilterChain(xdsListener *listenerv3.Listener, irL
 		}
 	}
 
+	if http3Listener {
+		mgr.CodecType = hcmv3.HttpConnectionManager_HTTP3
```

Comment:
will auto work ? 
is there a way for us to support http3 but also support something else in case the client doesnt support it ?

Reply:
Hi Arko, If HTTP3 is not supported then the other listener will be used. We have created a mirror UDP listener and the connection would be upgraded to HTTP3 according to the alt-svc header. So even if HTTP3 is not supported HTTP/HTTP2 will be used